test/jest-puppeteer-vuetify/src/jest-xpaths.js

```diff
@@ -1,7 +1,7 @@
 import 'expect-puppeteer';
```

Comment:
I think you can actually remove the dependency on `expect-puppeteer`. It adds some puppeteer bindings to the jest expect API, so you can do stuff like `await expect(page).toClick(aButton);`. We never actually use expect anywhere in this module, even though `jest-xpaths` adds bindings that are stylistically similar.

I think having a peer dependency on `expect-puppeteer` is sufficient.